mwm-3-kotlin-1/demo/app/src/main/java/com/mwm/demo_kotlin/java_interoperability/Interoperability.kt

```diff
@@ -0,0 +1,16 @@
+package com.mwm.demo_kotlin.java_interoperability
+
+import android.util.Log
+
+class Interoperability {
+    companion object {
+        const val TAG = "Interoperability"
+        @JvmField val myVar = 5
+        @JvmStatic fun myStaticFunctionInJava() {
+            Log.i("OtherActivity", "My static function in java in Interroperability")
+        }
+        fun myNotStaticFunctionInJava() {
+            Log.i("OtherActivity", "My not static function in java in Interroperability")

```

Comment:
In logs, Interoperability instead of Interroperability. I'm maybe missing something, because I do not understand your comment `// doesn't exist` in interr with 2 `r`

Reply:
I mean it's not accessible without using companion object.
I changed the commentary.